CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 * Prevent removal of the `broker` process role from KRaft mixed-nodes that have assigned partition-replicas
 * Improve broker scale-down prevention to continue in reconciliation when scale-down cannot be executed
 * Added support for Tiered Storage by enabling the configuration of custom storage plugins through the Kafka custom resource.
+* Create classes to represent the trust set and identity for client authentication when creating clients using TLS in operators.
```

Comment:
Not sure this needs a CHANGELOG record as it is completely internal implementation detail, or?

Reply:
Yeah I can remove it if it's not required

Reply:
Yes I agree, it doesn't need to be in the CHANGELOG.